doc/devel/documenting_mpl.rst

```diff
@@ -51,17 +52,86 @@ requirements that are needed to build the documentation. They are listed in
 
 * Sphinx>=1.3, !=1.5.0, !=1.6.4, !=1.7.3
 * colorspacious
-* IPython
+* IPython<7.0.0
 * numpydoc>=0.4
 * Pillow
 * sphinx-gallery>=0.1.13
+* LaTeX
 * graphviz
 
 .. note::
 
-  * You'll need a minimal working LaTeX distribution for many examples to run.
-  * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a Python package,
-    and needs to be installed separately.
+  * You'll need a minimal working `LaTeX <https://www.latex-project.org/>`_ distribution for many examples in the documentation to run.
+  * `Graphviz <http://www.graphviz.org/>`_ is not a Python package,
+    and needs to be installed separately. 
+
+
+Creating a development environment
+----------------------------------
+
+It's suggested that you create a separate python environment for working on matplotlib. You can do this using `conda <https://conda.io/docs/user-guide/tasks/manage-environments.html>`_ or `virtualenv <https://virtualenv.pypa.io/en/stable/>`_. You'll need to use python3 to work on the most current version of Matplotlib. 
```

Comment:
I'd rather consolidate this with what's already in contributing.rst rather than duplicating the same info in two places with two slightly different variants.

Reply:
linking is fine too.